tests/test_cli.py

```diff
@@ -20,6 +25,51 @@ def context():
     return ctx().obj["context"]
 
 
+mock_gui_main = """from .app import show
```

Comment:
The problem is that in order to test how pyblish-base is calling gui, ei. ```sys.executable, "-m", package + ".__main__"```, we need it to be a package.
Or maybe I'm missing something?